### ISSUE TITLE
Limited insight given on implementation (trying to reproduce results on other tasks etc.).

### ISSUE THREAD

**Issue by kostagiolasn** [lsw9021/MASS#22]
- You mention that there is a possibility of training a torque-only actuated model. Could you please walk me through the process of doing this as well as pointing me to the necessary data for this?
- How where the .xml files (human and muscle) and .obj files created? Were they produced from physical/empirical measurements or did you infer them in some way?
- Are the offsets included in the .xml files person-specific or generic (i.e. did you use only a single specific human for them?)
- Could you also explain to me what is the purpose of the reward_param field in the metadata.txt file? Are these values fine-tuned in some way? Running the code for the "running" task while keeping the reward_param constant produced negative results.

**Comment by clineil** [comment]
also wondering about the reward_param field 
training to walk works fine 
but i also produce negative results when training for running

**Comment by todayThursday** [comment]
> 还想知道 reward_param Field Training to Walk 效果很好，但我在跑步训练时也会产生负面结果

你好，我也是，我最近在训练跑步的模型，但无论怎么修改reward_param都没有训练出比较好的效果，我想知道您解决了这个问题吗，但是另一方面我又觉得奖励函数的权重参数其实并不影响训练模型的效果，在代码中，奖励函数只考虑了关节的位置误差和速度误差，并用末端执行器的误差对这两个进行缩放，按理说如果我们只考虑莫不运动的相似性，并不需要在添加其他的效果，只是神经网络对其训练的结果，那为什么不管怎么感觉run模型都没有很好的结果

**Comment by todayThursday** [comment]
> also wondering about the reward_param field training to walk works fine but i also produce negative results when training for running

I have been training the running model recently, but no matter how to modify reward_param, the training effect is not good, I want to know if you have solved this problem, but on the other hand, I think the weight parameter of the reward function does not affect the effect of the training model, in the code, The reward function only considers the position error and the velocity error of the joint, and scales these two with the error of the end-effector. Logically, if we only consider the similarity of the motion, and do not need to add other effects, just the results of the training of the neural network, then why does the run model not have good results no matter how it is felt
